apps/agent/src/renderer/styles.css

```
@@ -1,342 +1,1682 @@
+  color: var(--castro-gold-2);
+  box-shadow: 0 0 0 4px rgba(201, 166, 101, 0.14);
+}
+.ac-life-step.active .ac-life-dot::before {
+  content: "";
+  position: absolute;
+  inset: -4px;
+  border-radius: 999px;
+  border: 1px solid var(--castro-gold);
+  animation: acPulse 1.6s ease-out infinite;
+}
+@keyframes acPulse {
```

Stylelint (17.11.1)

[error] Expected keyframe name "acPulse" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)